package.json

```diff
@@ -47,7 +47,7 @@
     "twitter-api-v2": "^1.14.2",
     "typewriter-effect": "^2.20.1",
     "usehooks-ts": "^2.9.1",
-    "wagmi": "^0.8.10"
+    "wagmi": "^0.12.12"
```

Comment:
whoa BIG upgrade 🏗️ 

are we sure there aren't any unintended side-effects?

Reply:
this is the version that is in reserve-ui repo, the biggest change was the removal of `allChains` function. I replaced it with own array of acceptable chains, mainly `mainnet, goerli, polygon, mumbai`